src/Modules/Settings/Dnn.PersonaBar.Prompt/Services/CommandController.cs

```diff
@@ -16,20 +16,61 @@
 using DotNetNuke.Services.Exceptions;
 using DotNetNuke.Services.Localization;
 using DotNetNuke.Services.Log.EventLog;
+using DotNetNuke.Entities.Portals;
 
 namespace Dnn.PersonaBar.Prompt.Services
 {
     [MenuPermission(MenuName = "Dnn.Prompt")]
     [RequireHost]
-    public class CommandController : ControllerBase
+    public class CommandController : ControllerBase, IServiceRouteMapper
     {
         private static readonly ILog Logger = LoggerSource.Instance.GetLogger(typeof(CommandController));
         private static readonly string[] BlackList = { "smtppassword", "password", "pwd", "pass", "apikey" };
 
-        [HttpGet]
-        public HttpResponseMessage List()
```

Comment:
WHy this method have been removed?

Reply:
The "List" method was never used and superceded by the List command that passed through the main WebAPI method. I removed it to remove an unused endpoint.